charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml

```diff
@@ -44,6 +44,7 @@ spec:
   logFormat: {{ .Values.thanosRuler.thanosRulerSpec.logFormat | quote  }}
   logLevel:  {{ .Values.thanosRuler.thanosRulerSpec.logLevel | quote  }}
   retention: {{ .Values.thanosRuler.thanosRulerSpec.retention | quote  }}
+  evaluationInterval: {{ .Values.thanosRuler.thanosRulerSpec.evaluationInterval | quote }}
```

Comment:
If the default is not set this should not be applied at all.

Reply:
Done, could you please check again, thanks